### PR TITLE
feat: 在对象存储的piecestorage增加bucket配置项 / add bucket config in piecestorage

### PR DESCRIPTION
**Pull request by LinZexiao**
## Linked Issue

close https://github.com/filecoin-project/venus/issues/5069

## Proposal Change

add bucket config in piecestorage
fix the interface to add S3 piecestorage
change the way to explain config about S3 piecestorage

### CODE REVIEW

File: cli/piece-storage.go [hunk 1]
```diff
@@ -109,6 +123,9 @@ var pieceStorageAddS3Cmd = &cli.Command{
 		if !cctx.IsSet("name") {
 			return fmt.Errorf("name is required")
 		}
+		if !cctx.IsSet("bucket") {
```
Comment (simlecode) [inline cli/piece-storage.go:126]: 可以换一种写法，参考：https://github.com/filecoin-project/venus-market/blob/master/cli/actor.go#L76

**Review by simlecode — COMMENTED** [review]
把 extern/venus 传上来了

File: cli/piece-storage.go (hunk 1 above)
Comment (LinZexiao) [reply cli/piece-storage.go:126]: nice

File: docs/zh/venus-market配置解释.md [hunk 2]
```diff
@@ -414,7 +414,16 @@ ReadOnly = true
 
 # 对象存储服务的入口
 # 字符串类型 必选
-EndPoint = "http://oss-cn-shanghai.aliyuncs.com/venus-market-test"
+# 支持单独的EndPoint（"oss-cn-shanghai.aliyuncs.com"）和完整的EndPoint Url（"http://oss-cn-shanghai.aliyuncs.com"）
+EndPoint = "oss-cn-shanghai.aliyuncs.com"
```
Comment (hunjixin) [inline docs/zh/venus-market配置解释.md:418]: 信息隐藏一下，找个文档的例子写进来

File: cli/piece-storage.go [hunk 3]
```diff
@@ -83,15 +83,32 @@ var pieceStorageAddS3Cmd = &cli.Command{
...
+			Name:     "bucket",
+			Aliases:  []string{"b"},
+			Usage:    "name of the S3 bucket",
+			Required: true,
+		},
+		// dir
+		&cli.StringFlag{
+			Name: "subdir",
+			Aliases: []string{
+				"d",
```
Comment (hunjixin) [inline cli/piece-storage.go:109]: 写成一行

Comment (LinZexiao) [reply cli/piece-storage.go:109]: ok

File: cli/piece-storage.go [hunk 4]
```diff
@@ -183,10 +195,10 @@ var pieceStorageListCmd = &cli.Command{
 
 		for _, storage := range storagelist.S3Storage {
 			w.Write(map[string]interface{}{
-				"Name":                storage.Name,
-				"Readonly":            storage.ReadOnly,
-				"Path or Enter point": storage.EndPoint,
-				"Type":                "S3",
+				"Name":           storage.Name,
+				"Readonly":       storage.ReadOnly,
+				"Path or Bucket": storage.EndPoint + "/" + storage.Bucket + "/" + storage.SubDir,
```
Comment (hunjixin) [inline cli/piece-storage.go:200]: Path or Bucket.这个名称对吗

**Review by hunjixin — APPROVED** [review]

**Pull request merged**